pkg/reconciler/common/prune.go

```diff
@@ -388,12 +388,12 @@ func createCronJob(ctx context.Context, kc kubernetes.Interface, cronName, targe
 func generatePruneConfigPerNamespace(pru *pruneConfigPerNS, ns string) string {
 	var keepCmd string
 	if pru.config.Keep != nil {
-		keepCmd = "--keep=" + fmt.Sprint(*pru.config.Keep)
+		keepCmd = "--keep=" + fmt.Sprint(*pru.config.Keep) + ";"
```

Comment:
`:` is required at the end of a command.
here is keep is not nil we add it  like `--keep=2 ;`
and then next line if keepsince is not nill we add `--keep=2 ;--keep-since=300;`
where the actual command should have `--keep=2 --keep-since=300 ;`

Reply:
My understanding is the string will be passed as space-separated args wherever `;` is present in that case the final result will be the same. Isn't that right?

Reply:
here we are just generating the configs, actual code generation is happening at [here](https://github.com/tektoncd/operator/blob/da5694c9d27d2b411cbd944f9d4ddfc9a76c9b27/pkg/reconciler/common/prune.go#L253), please do add test for this.

Reply:
ok sure